nvme.c

```diff
@@ -6757,7 +6776,7 @@ static int gen_dhchap_key(int argc, char **argv, struct command *command, struct
 	if (!raw_secret)
 		return -ENOMEM;
 	if (!cfg.secret) {
-		if (getrandom(raw_secret, cfg.key_len, GRND_NONBLOCK) < 0)
+		if (getrandom_bytes(raw_secret, cfg.key_len) < 0)
 			return errno;
```

Comment:
I think we need to do a 'return -errno' here.

Reply:
hum I didnt notice that, I think you are right, since we are returning like above ( return -ENOMEM; ) for errors